src/full/Agda/TypeChecking/Serialise/Base.hs

```diff
@@ -384,17 +384,17 @@ icodeN :: forall t. ICODE t (IsBase t) => Currying (Domains t) (S Int32) =>
           All EmbPrj (Domains t) =>
           Int32 -> t -> Arrows (Domains t) (S Int32)
 icodeN tag _ =
-  currys (Proxy :: Proxy (Domains t)) (Proxy :: Proxy (S Int32)) $ \ args ->
-  icodeNode . (tag :) =<< icodeArgs (Proxy :: Proxy t) args
+  currys (Proxy :: Proxy (Domains t)) (Proxy :: Proxy (S Int32))
+    $ ((icodeNode . (tag :)) <=< icodeArgs (Proxy :: Proxy t))
```

Comment:
I don't think using point-free style here improves readability, in fact quite the opposite.